static/app/components/events/eventVitals.tsx

```diff
@@ -135,7 +135,7 @@ function EventVital({event, name, vital}: EventVitalProps) {
         <Name>{vital.name ?? name}</Name>
         <ValueRow>
           {failedThreshold ? (
-            <FireIconContainer size="sm">
+            <FireIconContainer data-test-id="threshold-failed-warning" size="sm">
```

Comment:
we could add a `data-test-id` to the icon and maybe check if on hover, the tooltip is rendered :)

Reply:
I'm indifferent. This test is functionally equal right now, so would prefer to follow up with that change, than try and make it for this now 😅